WooCommerce/Classes/ViewRelated/Dashboard/Settings/In-Person Payments/Payments Menu/SimplePaymentsMigrationView.swift

```diff
@@ -0,0 +1,74 @@
+import SwiftUI
+
+/// Shows information about the migration from simple payments to order creation in Menu tab > Payments > Collect Payment.
+struct SimplePaymentsMigrationView: View {
+    let addCustomAmount: () -> Void
+
+    var body: some View {
+        VStack(spacing: Layout.defaultVerticalSpacing) {
+            VStack(spacing: Layout.verticalSpacingBetweenElements) {
+                Image(uiImage: .bell)
+                Text(Localization.title)
+                    .titleStyle()
+                    .bold()
+                    .fixedSize(horizontal: false, vertical: true)
+            }
+            VStack(spacing: Layout.verticalSpacingBetweenElements) {
+                Text(.init(Localization.subtitle))
```

Comment:
Is the .init needed?
```suggestion
                Text(Localization.subtitle)
```

Reply:
Yes, `.init` is required to show markdown in the label (`*and*` as italic). I added a comment about this in https://github.com/woocommerce/woocommerce-ios/pull/12400/commits/f18bd8b50a9a80c9e86f33fb25dc9f0612a39bde.

Reply:
Good to know! I haven't had the chance to use markdown yet :D